lib/Runtime/Library/JavascriptLibraryBase.h

```diff
@@ -173,10 +173,7 @@ namespace Js
 
         JavascriptFunction* defaultAccessorFunction;
         JavascriptFunction* stackTraceAccessorFunction;
-        JavascriptFunction* throwTypeErrorAccessorFunction;
-        JavascriptFunction* throwTypeErrorCallerAccessorFunction;
-        JavascriptFunction* throwTypeErrorCalleeAccessorFunction;
-        JavascriptFunction* throwTypeErrorArgumentsAccessorFunction;
+        JavascriptFunction* throwTypeErrorRestrictedPropertyAccessorFunction;
```

Comment:
heads up, this will break the Edge binary consistency check until we get a new Edge build that includes this change